### ISSUE TITLE
Enabling theme per directions will cause qutebrowser 1.13.1 to error out after 3603 seconds of operation

### ISSUE THREAD

**Issue by duckunix** [dracula/qutebrowser#8]
Lots of debugging info over at https://github.com/qutebrowser/qutebrowser/issues/5656.

TL;DR of it is there is a new change in 1.13.1 which after enabling this theme (which is great, btw), the browser will crash just over an hour of operation.  Does not matter if I am using the browser, of if the browser is totally idle.

To repoduce:
install qutebrowser from git, install via the `mkvenv.sh` script, edit config.py per the instructions from this repo, fire it up.  3603-3606 seconds later, browser crashes.  Remove `~/.local/share/qutebrowser/webengine`, disable the inclusion of the dracula theme, fire the browser up, let it run for hours.

Please let me know what other info you might need.
